bugalink-frontend/hooks/useIndividualRides.ts

```diff
@@ -5,7 +5,7 @@ export default function useIndividualRides(individualRideId) {
   // TODO: once we have local sessions, we can extract the user id from the session
```

Comment:
There is already a way to extract the session with `useSession()`, in case you need access to the logged user (`user_id`, `driver_id`, `passenger_id` or personal data). Check other hooks